dev-util/intel-graphics-compiler/intel-graphics-compiler-1.0.9636.ebuild

```diff
@@ -0,0 +1,110 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_BUILD_TYPE="Release"
+LLVM_MAX_SLOT="13"
+MY_PN="igc"
+MY_P="${MY_PN}-${PV}"
+#PYTHON_COMPAT=( python3_{8..10} )
+PYTHON_COMPAT=( python3_10 )
+
+inherit cmake flag-o-matic llvm python-any-r1
+
+DESCRIPTION="LLVM-based OpenCL compiler for OpenCL targetting Intel Gen graphics hardware"
+HOMEPAGE="https://github.com/intel/intel-graphics-compiler"
+COMMIT_vc="8ee879314584e6630688b0a3b290d065dcabb383"
+COMMIT_ST="df2aad68b98279412494a6d449bd71b6756e699b"
+COMMIT_SH="eddd4dfc930f1374a70797460240a501c7d333f7"
+SRC_URI="https://github.com/intel/${PN}/archive/${MY_P}.tar.gz -> ${P}.tar.gz
+		https://github.com/intel/vc-intrinsics/archive/${COMMIT_vc}.tar.gz -> vc-intrinsics-${COMMIT_vc}.tar.gz
+		https://github.com/KhronosGroup/SPIRV-Tools/archive/${COMMIT_ST}.tar.gz -> SPIRV-Tools-${COMMIT_ST}.tar.gz
+		https://github.com/KhronosGroup/SPIRV-Headers/archive/${COMMIT_SH}.tar.gz -> SPIRV-Headers-${COMMIT_SH}.tar.gz"
+S="${WORKDIR}/${PN}-${MY_P}"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="debug"
+
+DEPEND="
+	dev-libs/opencl-clang:${LLVM_MAX_SLOT}=
+	sys-devel/llvm:${LLVM_MAX_SLOT}=
+"
+
+RDEPEND="${DEPEND}"
+
+BDEPEND="
+	${PYTHON_DEPS}
+	>=sys-devel/lld-${LLVM_MAX_SLOT}
```

Comment:
You add a path to the LLVM_SPIRV-Translator below, but in that case, you also have to depend on that ebuild here. Otherwise it might not be installed.